frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -134,6 +134,9 @@ class PlaneController extends React.PureComponent<Props> {
     // catchers (only necessary for HammerJS). We should refactor the
     // InputMouse handling so that this is not necessary anymore.
     // See: https://github.com/scalableminds/webknossos/issues/3475
+
+    // const tdId = `inputcatcher_${OrthoViews.TDView}`;
+    // Utils.waitForElementWithId(tdId).then(() => {
```

Comment:
Same, can this be removed?

Reply:
Whoops, I thought I had removed dead code, but apparently I was wrong. Should be cleaned up now!